Jenkinsfile

```diff
@@ -313,7 +313,7 @@ pipeline {
                         dockerfile {
                             filename 'Dockerfile.mockbuild'
                             dir 'utils/rpms/packaging'
-                            label 'docker_runner'
+                            label 'docker_ce_runner'
```

Comment:
This part of the change is a hack for the docker builders which shouldn't be needed after todays shutdown and will need to be backed out of master, so for 2.0 can we wait until later and land this PR without this component of the change.